### DISCUSSION TITLE
Week 5- Discussion & project ideas

### DISCUSSION THREAD

**Discussion by kassyoveson** [USU-ITLS-x265-23-summer/class-repo#113] [General]
**FREECODE CAMP**
**Likes:**
 I liked the registration form because it taught how HTML and CSS coding correspond with each other. I like how it gave hints if you got the answer wrong. It showed me where to look in the code to fix it.

**Dislikes:** 
It was hard to complete without looking up code. It didn’t always say if the code you were using was supposed to be HTML or CSS. I had to learn what they were asking based on if it was a ‘make this look better question’ or ‘add a line’. Some steps were hard because they wanted you to add code into 6 lines and you had to find the lines and hope you didn’t mess up on one of them. Sometimes I will transpose a letter, so I eventually learned to just copy and paste the code as needed.

**Changes:**
 I would like to see them tell us exactly what code language they want us to use for the problem. Towards the end, I understood. The second time doing this made it easier. I would also space out the code better, so it is easier to read and more organized. A bigger font might help too. I wish it had better explanations as to why you are changing things and adding it to the page. 

**FLEXBOX FROGGY**
**Likes:**
I liked how it was easy to understand. It gave the codes at the beginning and started with simple to understand instructions. It explained different terms and how to use them, which was nice. 

**Dislikes:** 
Not much to dislike about the game.
 
**Changes:** 
Make it longer with more levels. Maybe add more sound to it or a timer to see if you can beat your best score. 

**PROJECT IDEAS:**
1- Portfolio website with links to past projects to emphasize my degree. I would be able to start the outline with html and then design different creative  aspects with CSS to make it look better. I wonder if there is a way to track who views my portfolio page?
2-  School Lunch Count website with checkboxes and input boxes to do daily lunch counts and totals online. I would use the combination of html, java and CSS to write the code and make it look pretty.  I would need to figure out how to receive the data in an easy-to-read document. I may use this in combination for my capstone project.

**Comment by metasean** [comment]
Your comments about FCC mirror a lot of my own.  When you have time, going through a project a second time really helps you demystify why you're doing the different steps.

> a way to track who views my portfolio page

You could do this with something like Google Analytics

> School Lunch Count website with checkboxes and input boxes to do daily lunch counts and totals online

You could do the interactivity with JavaScript.  If you're trying to capture the data over time, or from multiple people, the you'd need a backend, and we won't be covering backends in this course.  However, if all the data is being input at the same time, and you're just looking to calculate, based on that data, then you would only need JavaScript.  You could then "print the page" to pdf.

fyi: `Java !== JavaScript` (just like `Car !== Carpet`)
